guides/introduction/community.md

```diff
@@ -1,17 +1,26 @@
 # Community
 
-## Elixir Forum
+Although the Elixir community is still rather young, its popularity is growing on quite significan pace.
```

Comment:
Although Elixir community is still quite young, its popularity is growing in a very significant pace.